### PR TITLE
Describe consumer thread-safety

### PR DESCRIPTION
**Pull request by ecksun**
A draft of some small improvements detailing thread safety of the KafkaConsumer.

This addresses issue #1105 using the details from that dicussion

### CODE REVIEW

**Comment by jeffwidman** [comment]
Thanks for starting this. 

There are multiple misspellings, I suggest you run this through a spell checker.

File: README.rst [hunk 1]
```diff
@@ -110,6 +110,16 @@ for more details.
...
 ...     producer.send('foobar', b'msg %d' % i)
 
+Thread safety
+*************
+
+The KafkaProducer can be used across threads without issue, unlike the
+KafkaConsumer which cannot.
+
+While it is possible to use the KafkaConsumer in a thread-local manner using
+multithreading is recommended, as is the case in general in python when it
```
Comment (dpkp) [inline README.rst]: I think you mean multiprocessing here.

**Comment by dpkp** [comment]
Yes -- producer is intended to be thread-safe, for better or for worse. I agree that multiprocessing is the better mode for general concurrency.

**Comment by dpkp** [comment]
Merged manually w/ fixups as `97261f4`

**Pull request closed without merging**